cl_manycore/libraries/bsg_manycore_cuda.cpp

```diff
@@ -30,11 +36,14 @@ static int hb_mc_get_x(tile_t *tiles, uint8_t *x_list, uint32_t num_tiles) {
 		return HB_MC_FAIL;
 	}
 	for (int i = 0; i < num_tiles; i++) {
-		x_list[i] = tiles[i].x;
+		x_list[i] = tiles[i].coord.x;
```

Comment:
Please use the setter functions for `hb_mc_coordinate_t` -- hb_mc_coordinate_t is currently a struct but someday it might be a 64-bit integer and then code using the `.` syntax would break.

Reply:
Done, thanks!